projects/lyonel_feininger/index.md

```diff
@@ -0,0 +1,166 @@
+--- 
+authors: 
+  - "Jakob von Kietzell"
+  - "Julian Broocks"
+authors-url: 
+  - "https://github.com/hulianbrox"
+  - "https://github.com/jakobvk"
+layout: project
+title: "Lyonel Feininger"
+type: project
+---
+
+![Vogelwolke](splash.png)
+
+<h1> Deconstructing Master Artists </h1>
```

Comment:
Also the title is added by the system - see https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax